jbpm-db-scripts/src/main/resources/db/ddl-scripts/db2/db2-jbpm-schema.sql

```diff
@@ -764,6 +764,8 @@
     create index IDX_CMI_KSession ON ContextMappingInfo(KSESSION_ID);    
     create index IDX_CMI_Owner ON ContextMappingInfo(OWNER_ID);
     
+    alter table CorrelationKeyInfo add constraint IDX_CorrelationKeyInfo_name unique (name);
```

Comment:
It should be placed above, around line 500 for coherence

Reply:
done for all scripts